### PR TITLE
Remove our proprietary protocol, standardize on the language service protocol.

### PR DESCRIPTION
**Pull request by rictic**
- [x] CHANGELOG.md has been updated

Now that we've got workflows for Atom and Sublime using the LSP we don't need this old code, and keeping compatibility with it is holding us back, e.g. in reacting to changes in polymer.json.

---
This change is [Reviewable](https://reviewable.io/reviews/polymer/polymer-editor-service/76)

### CODE REVIEW

File: src/local-editor-service.ts [hunk 1]
```diff
@@ -31,12 +30,11 @@ export interface Options extends AnalyzerOptions { polymerJsonPath?: string; }
  * has the same interface but is actually running a LocalEditorService in
```
Comment (valdrinkoshi) [inline src/local-editor-service.ts]: comment needs to be updated

Comment (rictic) [reply src/local-editor-service.ts]: +1, updated

File: CHANGELOG.md [hunk 2]
```diff
@@ -23,7 +26,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Warnings should now immediately be updated in response to any change, whether it happens in the editor or from external tools, like `bower install`.
 - Added `polymer-editor-service` as an npm binary, which speaks the [Language Server Protocol](https://github.com/Microsoft/language-server-protocol) over stdin/stdout. This makes it easier to get started integrated the editor service into a new editor.
 
-- Warnings should now immediately be updated in response to any change, whether it happens in the editor or from external tools, like `bower install`.
+- Removes our proprietary protocol in favor of unifying on the Language Server Protocol. Using the standard protocol lets us support more editors with less code.
```
Comment (valdrinkoshi) [inline CHANGELOG.md]: dupe of L15

File: CHANGELOG.md [hunk 3]
```diff
@@ -23,7 +26,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Warnings should now immediately be updated in response to any change, whether it happens in the editor or from external tools, like `bower install`.
 - Added `polymer-editor-service` as an npm binary, which speaks the [Language Server Protocol](https://github.com/Microsoft/language-server-protocol) over stdin/stdout. This makes it easier to get started integrated the editor service into a new editor.
 
-- Warnings should now immediately be updated in response to any change, whether it happens in the editor or from external tools, like `bower install`.
```
Comment (valdrinkoshi) [inline CHANGELOG.md]: 💯

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "polymer-editor-service",
-  "version": "1.5.0",
+  "version": "1.6.0",
```
Comment (valdrinkoshi) [inline package.json]: weird, it marks it as changed, but this is already the version in `master`

File: CHANGELOG.md (hunk 2 above)
Comment (rictic) [reply CHANGELOG.md]: lol, good catches! so many merge errors

File: package.json (hunk 4 above)
Comment (rictic) [reply package.json]: huh. well, rebasing seems to have solved it

**Pull request merged**